src/hooks/v1/transactor/PayV1ProjectTx.ts

```diff
@@ -23,7 +23,21 @@ export function usePayV1ProjectTx(): TransactorInstance<{
       !contracts?.TicketBooth ||
       !terminal?.version
     ) {
-      txOpts?.onDone?.()
+      let missingParam = !transactor
+        ? 'transactor'
+        : !projectId
+        ? 'projectId'
+        : !contracts?.TicketBooth
+        ? 'contracts.TicketBooth'
+        : !terminal?.version
+        ? 'terminal.version'
+        : null
+
+      txOpts?.onError?.(
+        new DOMException(
```

Comment:
`ErrorCallback` takes `DOMException`, I also didn't see it used anywhere else in the codebase. After looking around a bit: https://stackoverflow.com/questions/16142583/whats-the-difference-between-error-and-exception-in-javascript, Exception seemed like the right choice.

Reply:
Well there ya go! Don't think I've specifically encounted DOMException before. LGTM